docs/components/MultiSelect/MultiSelect.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Canvas, Meta } from "@storybook/addon-docs";
+import { useState } from "react";
+import { Divider } from "@jobber/components/Divider";
+import { Content } from "@jobber/components/Content";
+
+<Meta title="Components/Selections/MultiSelect/Docs" />
+
+# Multi Select
+
+The MultiSelect component gives our users the possibility to select multiple
+options from a menu.
+
+## Design & usage guidelines
+
+The goal of the MultiSelect component is to allow a user to check multiple
+options within a list of items.
+
+A user can click on the element to expand it and display the multiple options.
+The element will display the selected options or the placeholder text if none
+are selected.
+
+Every time an option is checked or unchecked, a behavior can be triggered using
+the `onOptionsChange` prop.
+
+Adjusting the
+[size](../?path=/story/components-selections-multiselect-web--sizes) of the
+MultiSelect can be used to help align with common adjacent elements such as
+[InputText](../?path=/docs/components-forms-and-inputs-inputtext-docs--page) or
+[Select](../?path=/docs/components-selections-select--select).
+
+## Accessibility
+
+- The MultiSelect trigger element contains a downward-pointing-arrow to hint
+  that it can be expanded
```

Comment:
```suggestion
  that it can be expanded.
```

Only marked this one, but all of these bulleted items are sentences and (imho) should end with periods.

Reply:
oof - I strongly dislike periods at the end of bullet points.  I can however, see the other opinion, as usually our docs have full sentence bullet points, which should call for a period (according to Grammarly 👀 )
I've actively been taking them out this whole time but will modify this moving forward. Then, because everything should be the same, I will go back after docs are migrated to make sure everything follows the same pattern.

Reply:
I'm open to a wider discussion! But, these aren't bullet points in the sense of a list of things (where you'd have a semicolon).... They tend to be full sentences gathered together. Use your discretion and we can adjust in time. :)